css/app/datepicker.css

```diff
@@ -89,6 +89,14 @@
 	border-left: 0;
 }
 
+.todayHilight {
```

Comment:
Also please use class naming like: `.highlight-today` and `.highlight-weekend`. :)